onnxruntime/contrib_ops/cuda/sparse/sparse_attention_v1/sparse_attention_v1_bf16_m16_0_n64_0_d64_2_sm80.cc

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "contrib_ops/cuda/sparse/sparse_attention_v1/sparse_attention_common.h"
+
+namespace onnxruntime {
+namespace contrib {
+namespace cuda {
+namespace sparse_attention_v1 {
+
+// This file is generated by compile_sparse_attention.py using triton AoT compiler
```

Comment:
Do we need to keep these .cc files in the codebase if they can be generated?

Reply:
It is needed since triton generation is for current GPU. In the future, if triton are able to compile for any sm that is not current GPU compute compacity, then we do not need those files.